index.html

```diff
@@ -1,171 +1,435 @@
+  <link href="https://getbootstrap.com/docs/4.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous" />
+
+  <!-- CSS extras -->
+  <link href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" rel="stylesheet" />
+  <link href="https://getbootstrap.com/docs/4.1/assets/css/docs.min.css" rel="stylesheet" />
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400" />
+  <link rel="stylesheet" href="stylesheets/style.css" />
 </head>
-<body data-spy="scroll" data-target="#toc" class="book">
-<div class="container">
-  <div class="row" id="main-row">
-    <div id="header" class="col-sm-2 border-columns">
-      <div class="mx-auto text-center sticky-top">
-        <a href="/">
-          <img class="logo rounded mx-auto d-block sticky-top" src="https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/logo.png">
-        </a>
-      </div>
-      <nav id="toc" data-toggle="toc" class="toc sticky-top">
-        <ul class="nav navbar-nav">
-          <li><a class="nav-link active">Getting Started</a>
-            <ul class="nav navbar-nav">
-              <li><a class="nav-link" href="http://docs.killbill.io/latest/what_is_kill_bill.html">What is Kill Bill?</a></li>
-              <li><a class="nav-link" href="http://docs.killbill.io/latest/getting_started.html"><b>Quick start</b></a></li>
+
+<body>
+  <div class="container-fluid">
+    <div class="row flex-xl-nowrap">
+      <div class="col-12 col-md-3 col-xl-3 bd-sidebar">
+        <div class="mx-auto text-center sticky-top my-3">
+          <a href="/">
+            <img class="logo rounded mx-auto d-block sticky-top img-fluid"
+              src="https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/logo.png" />
+          </a>
+          <button class="btn btn-link bd-search-docs-toggle d-md-none p-0 ml-3 toggle-nav-btn" type="button"
+            data-toggle="collapse" data-target="#bd-docs-nav" aria-controls="bd-docs-nav" aria-expanded="false"
+            aria-label="Toggle docs navigation">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30" width="30" height="30" focusable="false">
+              <title>Menu</title>
+              <path stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-miterlimit="10"
+                d="M4 7h22M4 15h22M4 23h22"></path>
+            </svg>
+          </button>
+        </div>
+
+        <nav class="collapse bd-links" id="bd-docs-nav">
+          <div class="bd-toc-item active">
+            <a class="bd-toc-link main-link ml-2"> Getting started </a>
+
+            <ul class="nav bd-sidenav pl-3">
+              <li class="active bd-sidenav-active">
+                <a href="http://docs.killbill.io/latest/what_is_kill_bill.html">
```

Comment:
I'm wondering if we shouldn't switch all links to HTTPS?

BTW I'm also seeing mix-content warnings on your version:
![image](https://user-images.githubusercontent.com/51940/104813444-36a61e00-5801-11eb-9465-f71247836e93.png)